src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -83,7 +94,7 @@ private synchronized void startVideo(final int deviceNumber) throws IOException
         try {
             grabber.start();
         } catch (FrameGrabber.Exception e) {
-            throw new IOException("A problem occurred trying to start the frame grabber for device number " + deviceNumber, e);
+            throw new IOException("A problem occurred trying to start the frame grabber", e);
```

Comment:
No, but you have the information from above: `"IP Camera " + new URL(address).getHost()` and the device number. I think this will be helpful in issue messages that get posted.